guide/english/bootstrap/jumbotron/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Jumbotron
+---
+A jumbotron indicates a big grey box for calling extra attention to some special content or information.
+
+Use a <div> element with class jumbotron to create a jumbotron
+
+**Code Example:**
+
+`<div class="jumbotron">
+  <h1>Jumbotron</h1> 
+  <p>This is a jumbotron sample</p> 
+</div>`
+
+<div class="jumbotron">
```

Comment:
Lines 15-18: You don't need to add the same code outside of code formatting. Because Bootstrap is not included in here, this will have no effect. Please remove this one and the second example (lines 31-36) as well.